HW13/hw13.py

```diff
@@ -0,0 +1,24 @@
+class MyOwnIterator:
+
+    def __init__(self, data, start_index, end_index):
+        self.data = data
+        self.start_index = start_index
+        self.end_index = end_index
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        if self.start_index < self.end_index:
```

Comment:
what if end_index is passed as 100000? but I have only 5 elements in list?

what if I pass index -10 to start_index? 


Please think about all possible situation